lib/challenge.js

```diff
@@ -60,6 +92,19 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function pet(name, species, breed, noise) = {
```

Comment:
That equal sign (not an assignment operator; assignment operators always have a variable identifier to their left) between `... noise)` and `{` breaks what is otherwise a good constructor function.